h2-storage/src/main/java/com/cgi/bonnie/h2storage/user/H2UserRepository.java

```diff
@@ -3,8 +3,15 @@
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface H2UserRepository extends CrudRepository<AssemblyUser, Long> {
 
+    public AssemblyUser findByName(String username);
+
+    public AssemblyUser findByEmail(String email);
```

Comment:
Yes, that's right, and also, currently the application identifies the users based on username, which is not the best attempt (to say the least). The findByEmail method is not used yet, but should be pretty easy to change to identify users by email address using this method instead of findByName (and I think that is what we should do, as later it will be easier to bind users to social media accounts like FB or Google)